bin/cron/analyze_hoc_activity

```diff
@@ -80,6 +80,10 @@ def main
   hoc_year = DCDO.get("hoc_year", 2017)
 
   # Compute the number of hoc events (the grand total and broken down by company and country).
+  # Note that these queries use the "DB" connection, which is set in lib/cdo/properties.rb to PEGASUS_DB
+  # PEGASUS_DB is defined in lib/cdo/db.rb using sequel with connections to the pegasus writer and reader.
+  # So, even though most of the script uses the reader connections (eg, DASHBOARD_DB_READER),
+  # the "DB" connection used here does include a connection to the writer.
```

Comment:
If we're just doing SELECT queries with the `DB` connection here, should we just change `DB[:forms].where` to `PEGASUS_DB_READER[:forms].where` so this script is consistently reading from the same place?

Reply:
That makes sense to me -- let me try and switch and confirm everything works smoothly on the clone.